<ui version="4.0" >
 <class>PlasmagikDesignedWidget</class>
 <widget class="QWidget" name="PlasmagikDesignedWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>486</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="mainWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>481</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>3</hsizetype>
     <vsizetype>3</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>460</width>
     <height>400</height>
    </size>
   </property>
   <property name="autoFillBackground" >
    <bool>true</bool>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="currentIndex" >
    <number>2</number>
   </property>
   <widget class="QWidget" name="page_3" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>9</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="firstLabel" >
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;">Hello,&lt;br />with this dialog you are going to magically package your&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;">new exciting creation.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;">Your creation will need to be in an empty folder, containing just the files&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;">you want to be packaged. Please note that the &lt;span style=" font-weight:600; color:#000000;">WHOLE &lt;/span>&lt;span style=" color:#000000;">directory,&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; color:#000000;">with all its content, will be packaged.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;">If you are going to use a custom licence, please&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;">remember to add a file called '&lt;span style=" font-weight:600;">COPYING&lt;/span>'  in the top level of the directory&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;">you are going to package. Note that this file existance will be checked,&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;">and the filename must be uppercase (the check is  case-sensitive).&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="textFormat" >
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>71</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>9</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>1</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Fill all the fileds:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>51</width>
           <height>7</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>Required fields:</string>
           </property>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="0" >
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QGridLayout" >
                <property name="margin" >
                 <number>0</number>
                </property>
                <property name="spacing" >
                 <number>6</number>
                </property>
                <item row="0" column="0" >
                 <widget class="QLabel" name="plasmoidNameLabel" >
                  <property name="text" >
                   <string>Name:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" >
                 <widget class="KLineEdit" name="plasmoidName" />
                </item>
                <item row="1" column="0" >
                 <widget class="QLabel" name="label_6" >
                  <property name="text" >
                   <string>Author:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1" >
                 <widget class="KLineEdit" name="plasmoidAuthor" />
                </item>
                <item row="2" column="0" >
                 <widget class="QLabel" name="plasmoidVersionLabel" >
                  <property name="text" >
                   <string>Version:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1" >
                 <widget class="KRestrictedLine" name="plasmoidVersion" />
                </item>
                <item row="3" column="0" >
                 <widget class="QLabel" name="label_10" >
                  <property name="text" >
                   <string>Licence:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1" >
                 <widget class="QComboBox" name="plasmoidLicence" >
                  <item>
                   <property name="text" >
                    <string/>
                   </property>
                  </item>
                  <item>
                   <property name="text" >
                    <string>GPL</string>
                   </property>
                  </item>
                  <item>
                   <property name="text" >
                    <string>LGPL</string>
                   </property>
                  </item>
                  <item>
                   <property name="text" >
                    <string>BSD</string>
                   </property>
                  </item>
                  <item>
                   <property name="text" >
                    <string>MPL</string>
                   </property>
                  </item>
                  <item>
                   <property name="text" >
                    <string>Creative Commons 3.0 BY</string>
                   </property>
                  </item>
                  <item>
                   <property name="text" >
                    <string>Creative Commons 3.0 BY-SA</string>
                   </property>
                  </item>
                  <item>
                   <property name="text" >
                    <string>Creative Commons 3.0 BY-ND</string>
                   </property>
                  </item>
                  <item>
                   <property name="text" >
                    <string>Creative Commons 3.0 BY-ND-NC</string>
                   </property>
                  </item>
                  <item>
                   <property name="text" >
                    <string>Creative Commons 3.0 BY-NC-SA</string>
                   </property>
                  </item>
                  <item>
                   <property name="text" >
                    <string>Custom licence...</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="4" column="0" >
                 <widget class="QLabel" name="label_20" >
                  <property name="text" >
                   <string>Content type:  </string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1" >
                 <widget class="QComboBox" name="plasmoidLanguage" />
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>25</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>Optional fields:</string>
           </property>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="0" >
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QGridLayout" >
                <property name="margin" >
                 <number>0</number>
                </property>
                <property name="spacing" >
                 <number>6</number>
                </property>
                <item row="0" column="0" >
                 <widget class="QLabel" name="label_2" >
                  <property name="text" >
                   <string>Homepage: </string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" >
                 <widget class="KLineEdit" name="plasmoidHomepage" />
                </item>
                <item row="1" column="0" >
                 <widget class="QLabel" name="label_4" >
                  <property name="text" >
                   <string>Email: </string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1" >
                 <widget class="KLineEdit" name="plasmoidEmail" />
                </item>
                <item row="2" column="0" >
                 <widget class="QLabel" name="label_9" >
                  <property name="text" >
                   <string>PNG Icon:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1" >
                 <widget class="KUrlRequester" name="plasmoidIcon" />
                </item>
                <item row="3" column="1" >
                 <widget class="KIconButton" name="kiconbutton" />
                </item>
                <item row="3" column="0" >
                 <widget class="QLabel" name="label" >
                  <property name="text" >
                   <string>Or, choose a system icon:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_2" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>9</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="plasmoidDescriptionLabel" >
         <property name="text" >
          <string>Add a simple description:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KLineEdit" name="plasmoidDescription" />
       </item>
       <item>
        <widget class="QLabel" name="releaseNotesLabel" >
         <property name="text" >
          <string>Here you can add release notes for your creation:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KTextEdit" name="plasmoidNotes" />
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page4" >
    <widget class="QWidget" name="layoutWidget" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QFrame" name="frame_3" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_8" >
             <property name="text" >
              <string>Please select the folder which contains your sources:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KUrlRequester" name="sourcePathRequester" />
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>21</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QFrame" name="firstFileFrame" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="firstFileLabel" >
             <property name="text" >
              <string>Please choose the file that should be loaded first:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KUrlRequester" name="firstFile" />
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>21</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QFrame" name="frame_4" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_11" >
             <property name="text" >
              <string>Please choose where do you want to save your plasmoid:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KUrlRequester" name="destFile" />
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>21</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KRestrictedLine</class>
   <extends>KLineEdit</extends>
   <header>krestrictedline.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kicondialog.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kiconbutton</sender>
   <signal>iconChanged(QString)</signal>
   <receiver>plasmoidIcon</receiver>
   <slot>setPath(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>241</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel" >
     <x>237</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
